swagger/definitions/resource/webhook.yml

```
@@ -20,8 +20,20 @@ properties:
         "contact_created",
         "contact_updated",
         "message_created",
+        "message_incoming",
```

YAMLlint (1.38.0)

[error] string value is redundantly quoted with any quotes

(quoted-strings)